docroot/sites/all/themes/ilr_theme/scss/components/_subsite_header.scss

```diff
@@ -119,14 +119,26 @@
       }
     }
   }
+
   &.ilr-at-70 #header-region {
-    background: $brown url("../images/early-years-nav-bg.jpg") no-repeat center bottom;
+    background: $brown;
     border-bottom-color: $darker-red;
     border-bottom: none;
     a {
       color: $gray-medium;
     }
   }
+
```

Comment:
We should probably pull out the overrides for Worker Institute then as well and create a separate sass file for it.